src/datasources/cache/cache.first.data.source.ts

```diff
@@ -47,15 +51,19 @@ export class CacheFirstDataSource {
   ): Promise<T> {
     const cached = await this.cacheService.get(cacheDir);
     if (cached != null) {
-      winston.debug(`[Cache] Cache hit: [${cacheDir.key}, ${cacheDir.field}]`);
+      this.loggingService.debug(
+        `[Cache] Cache hit: [${cacheDir.key}, ${cacheDir.field}]`,
```

Comment:
Something to consider: we can restructure log too. Maybe having something like:

```javascript
{
  "type": "cache_hit",
  "key": <key>,
  "field": <field>,
}
```

(no need to tackle this in this PR)

Reply:
Yes, good point, changed in [e71d09d](https://github.com/safe-global/safe-client-gateway-nest/pull/310/commits/e71d09d04b82581fa7504f3f5361f920c54b8dd3)